platform/src/main/java/org/stellar/anchor/platform/controller/sep/SepControllerConfig.java

```diff
@@ -0,0 +1,16 @@
+package org.stellar.anchor.platform.controller.sep;
+
+import org.springframework.beans.propertyeditors.StringTrimmerEditor;
+import org.springframework.web.bind.WebDataBinder;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.InitBinder;
+
+@ControllerAdvice
```

Comment:
Does this happen to platform API as well?

Reply:
At the moment, no. I'll add the config anyway to the platform and custody controllers anyway to prevent future issues.